projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -103,11 +107,11 @@ const Admin = () => {
 						{ shouldShowAdvancedPlanNudge && <AdvancedUpsellNotice /> }
 						<InstagramNotice onUpgrade={ onUpgradeToggle } />
 						<SocialModuleToggle />
-						{ ! isUpdatingJetpackSettings && isModuleEnabled && isAutoConversionAvailable && (
-							<AutoConversionToggle />
+						{ isModuleEnabled && isAutoConversionAvailable && (
+							<AutoConversionToggle disabled={ isUpdatingJetpackSettings } />
```

Comment:
`isUpdatingJetpackSettings` is whether the module value is being updated, I think? Have we got an equivalent for `isRefreshingSocialSettings`?

Reply:
Yeah, we have, and that one is already considered in the toggle itself. I added this here to avoid the scenario where the `isRefreshingSocialSettings` finishes faster than the module toggle update, and there is that weird state where the module toggle is still disabled and updating, but the children are not. This way it always waits for the main toggle to be not disabled.